README.md

```diff
@@ -77,6 +77,11 @@ $ git lfs track "*.psd"
 more about this pattern syntax
 [here](https://git-scm.com/docs/gitattributes)).
 
+**Note**: the quotations surrounding the pattern are important. Otherwise
+the glob pattern will be interpreted first by your shell and expanded into matching
+existing files. Multiple individual file entries will be created in 
```

Comment:
I think we have some trailing whitespace on this line, which is causing the CI system to be upset. Could you remove it?